core/src/main/resources/xml/xsl/stub4testtool.xsl

```diff
@@ -16,6 +16,7 @@
 		- add the attribute useFixedValues with value true to all pipe, inputWrapper and outputWrapper elements SoapWrapperPipe
 		- stub the pipe element GetPrincipalPipe by a pipe element FixedResultPipe with attribute returnString set to tst9
 		- stub the pipe element IsUserInRolePipe by a pipe element EchoPipe
+		- add the attribute thenOnInput with the value 'cn=then,ou=stub,ou=LdapIsMemberOfPipe,dc=frankframework,dc=org' and attribute exceptionOnInput with value 'cn=exception,ou=stub,ou=LdapIsMemberOfPipe,dc=frankframework,dc=org' to all pipe elements LdapIsMemberOfPipe
```

Comment:
Ik vind het niet zo'n prettig idee als allerlei pipes hier nu hun eigen stubmechanisme gaan implementeren.
Het zou beter zijn dat op een meer generieke manier op te lossen.

Reply:
Feature request #1614 is aangemaakt om generieke stubbing te implementeren, deze gaat zolang op draft.